providers/implementations/encode_decode/decode_pem2der.c

```diff
@@ -106,7 +106,11 @@ static int pem2der_pass_helper(char *buf, int num, int w, void *data)
     return (int)plen;
 }
 
-static int pem2der_decode(void *vctx, OSSL_CORE_BIO *cin,
+/*
+ * pem2der_decode() doesn't care one bit about selections.  That's for the
```

Comment:
I am not sure this comment is going to mean much to anyone but the implementor of the function.

Reply:
Does this translate to.
The selection parameter is not used by this function because....

Reply:
> Does this translate to...

Yes.  I'll change to that